### PR TITLE
Fix for home-assistant/home-assistant/issues/3378

### PR DESCRIPTION
**Pull request by justweb1**
@balloob, This cannot be tested against Demo due to all values being set. This still needs to be tested. We may need @turbokongen to make some changes on backend to support this update.

### CODE REVIEW

**Comment by justweb1** [comment]
Fix for home-assistant/home-assistant#3378

File: src/state-summary/state-card-climate.html [hunk 1]
```diff
@@ -72,13 +72,12 @@
   computeTargetTemperature: function (stateObj) {
     var stateTemp = '';
 
-    if (stateObj.attributes.operation_mode === 'off') {
-      stateTemp = '';
-    } else if (stateObj.attributes.operation_mode === 'auto') {
+    if (stateObj.attributes.target_temp_low &&
+        stateObj.attributes.target_temp_high) {
```
Comment (turbokongen) [inline src/state-summary/state-card-climate.html]: Shouldn't we also check the operation mode? All three?

**Comment by pvizeli** [comment]
:+1:

**Comment by justweb1** [comment]
@turbokongen I have also updated more-infos to be displayed based on temp not operation mode. My concern is that we may not be able to display sliders at all if we are clearing out the temps on operation mode changes. I'm thinking that on the backend we set temperature_default, target_temp_low_default, and target_temp_high_default. We can allow these to be changed in the configuration. Then we check if an operation mode is set with no value change value to the default values.

**Pull request merged**